lib/temporal/activity/poller.rb

```diff
@@ -66,6 +66,11 @@ def poll_for_task
         client.poll_activity_task_queue(namespace: namespace, task_queue: task_queue)
       rescue StandardError => error
         Temporal.logger.error("Unable to poll activity task queue: #{error.inspect}")
+
+        Temporal.configuration.error_handlers.each do |handler|
```

Comment:
Let's get this under a more concise interface, something like `Temporal::ErrorHandler.handle(error, nil)`